pypeit/calibrations.py

```diff
@@ -645,6 +652,239 @@ def get_scattlight(self):
 
         return self.msscattlight
 
+    def make_slitless_pixflat(self):
```

Comment:
I think we want to move this out into a new class.  I also need to think through use of the cache.